<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PfmHomPage</class>
 <widget class="QDialog" name="PfmHomPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>950</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PermitNewBtn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>500</y>
     <width>150</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Permit New Hotel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ToNextBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>500</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>To Next Day</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LogoutBtn">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>500</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Log out</string>
   </property>
  </widget>
  <widget class="QLabel" name="d_TodayLabel">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>500</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TodayLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>500</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Today is</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
